### PR TITLE
Style

### PR DESCRIPTION
**Pull request by rhenwood39**
Style fixes in URModel, PopModel, EventStore

### CODE REVIEW

File: src/main/java/org/template/recommendation/URModel.java [hunk 1]
```diff
@@ -16,6 +17,7 @@
 import java.util.*;
 
 /** Universal Recommender models to save in ES */
+@AllArgsConstructor
 public class URModel {
 
     private transient static final Logger logger = LoggerFactory.getLogger(URModel.class);
```
Comment (jk2227) [inline src/main/java/org/template/recommendation/URModel.java]: https://projectlombok.org/features/Constructor.html

Based on this I feel like the logger might be included as part of the constructor..? O:

File: src/main/java/org/template/recommendation/PopModel.java [hunk 2]
```diff
@@ -12,14 +13,13 @@
...
 import scala.Tuple2;
-import scala.reflect.ClassTag;
-import scala.reflect.ClassTag$;
 
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+@AllArgsConstructor
```
Comment (jk2227) [inline src/main/java/org/template/recommendation/PopModel.java]: refer to comment in URModel

Comment (rhenwood39) [inline src/main/java/org/template/recommendation/PopModel.java]: From the link you pasted: 

> Static fields are skipped by these annotations.

Also, my Pop Model tests still work, and in them I don't pass anything related to logger into my PopModel constructors.

Comment (rhenwood39) [inline src/main/java/org/template/recommendation/PopModel.java]: If you think it's too confusing, I can de-lombok it.

Comment (jk2227) [inline src/main/java/org/template/recommendation/PopModel.java]: oh woops my bad i didn't read too carefully... let's keep the lombok then

**Pull request merged**